packages/debug/src/browser/debug-session.tsx

```diff
@@ -221,12 +221,12 @@ export class DebugSession implements CompositeTreeElement {
     get currentThread(): DebugThread | undefined {
         return this._currentThread;
     }
-    set currentThread(thread: DebugThread | undefined) {
+    setCurrentThread(forceUpdateSession: boolean, thread: DebugThread | undefined): void {
```

Comment:
This is a breaking change, if possible please try not to break the API, else a note will be required in the `changelog` for the next release in the `breaking changes` section.

Reply:
I have wrapped setCurrentThread in currentThread.